src/lib/connectors/water-service/batch-notifications.js

```diff
@@ -0,0 +1,24 @@
+const serviceRequest = require('../service-request');
+const config = require('../../../../config');
+
+const getBaseUrl = () => `${config.services.water}/batch-notifications`;
+
+const prepareReturnsReminders = (issuer, excludeLicences) => {
+  const url = `${getBaseUrl()}/prepare/returnReminder`;
+  return serviceRequest.post(url, {
+    body: {
+      issuer,
+      data: {
+        excludeLicences
+      }
+    }
+  });
+};
+
+const sendReturnsReminders = eventId => {
```

Comment:
I think I need to make a change to the water service so that the issuer is also sent with this request, and this is checked against the issuer of the prepare event above.